app/config/StringsConfig.scala

```diff
@@ -5,6 +5,6 @@ import com.typesafe.config.ConfigFactory
 class StringsConfig {
   val config = ConfigFactory.load("strings.conf")
 
-  val bundleLandingDescription = config.getString("bundleLanding.description")
+  val supportLandingDescription = config.getString("supportLanding.description")
```

Comment:
Is 'support landing page' the best terminology? The whole site is the support site so 'landing page' would seem to be enough. 

I don't feel too strongly about it though if you can't face changing it all again!

Reply:
I would try to keep the same pattern that we used in other pages e.g. `[name]-landing`, in this case probably the best name would be `bundle-landing`